### ISSUE TITLE
Mangled identifier generator generating invalid identifier

### ISSUE THREAD

**Issue by youdie323323** [MichaelXF/js-confuser#156]
**Describe the bug:**

Obfuscator generate invalid identifier.
Simply add "do" to reserved keywords and this issue will fix.

**Config and Small code sample**

Config:

```js
{
              target: 'browser',
              preset: 'low',

              stringCompression: false,
              stringConcealing: false,
              stringEncoding: false,
            
              globalConcealing: true,
              opaquePredicates: true,
              controlFlowFlattening: 0.25,

              identifierGenerator: 'mangled',
              hexadecimalNumbers: false,
              deadCode: false,
}
```

Code:

```js
// Too big, i cant provide it
```

**Expected behavior**

The program should work correctly.

**Actual behavior**

The error appears.
```
Uncaught SyntaxError: Unexpected token 'do' (at client.js:1:36358)
```

**Additional context**

Version: 2.0.0-alpha.4
![image](https://github.com/user-attachments/assets/a8b5f975-c37c-47a0-ac3f-1a35c1bc0b4c)

**Comment by MichaelXF** [comment]
Fixed in https://github.com/MichaelXF/js-confuser/commit/06d77666222bf15bab820b11fed5dcbe2bb2383a, will be released soon

**Comment by MichaelXF** [comment]
Released in [2.0.0-alpha.5](https://www.npmjs.com/package/js-confuser/v/2.0.0-alpha.5)
